components/mpas-ocean/src/Registry.xml

```diff
@@ -1023,6 +1052,10 @@
 					description="constant nondimensional salt transfer coefficient across the ice-ocean boundary layer"
 					possible_values="Any positive real number"
 		/>
+		<nml_option name="config_subglacial_runoff_mode" type="character" default_value="off"
+					description="Selects the mode in which subglacial runoff fluxes are implemented."
+					possible_values="'off', 'data'"
+		/>
```

Comment:
I think this needs to get moved up to the same place as the other config options.  I would put it as the first config options before the others you added.